packages/e2e-playwright/tests/e2e/card/bcmc/dualBranding.spec.ts

```diff
@@ -13,7 +13,9 @@ import { URL_MAP } from '../../../../fixtures/URL_MAP';
 test.describe('Bcmc payments with dual branding', () => {
     test.describe('Bancontact (BCMC) / Maestro brands', () => {
         test.describe('Selecting the Bancontact brand', () => {
-            test('should submit the bcmc payment', async ({ bcmc }) => {
+            test('#1a should submit the bcmc payment', async ({ bcmc, page }) => {
+                const paymentsRequestPromise = page.waitForRequest(request => request.url().includes('/payments') && request.method() === 'POST');
```

Comment:
I think this can be moved to unit test though, inspecting the correct brand sent in the payload data.

Reply:
Why not just do it here - since we're submitting the payload anyway?

Reply:
We can do it here, but i think it's better to keep the e2e test suite simple and easy (not testing too much implementation details). Plus it doesn't calculate the coverage if we put too much things into the e2e tests.

Reply:
I'll look into it in another PR